### DISCUSSION TITLE
Launch mode unavailable

### DISCUSSION THREAD

**Discussion by iamLazyCode** [microsoft/terminal#16111] [General]
is the launch mode to set terminal to open full screen removed ? if so how can i set it to full screen by default

all i see is size

![image](https://github.com/microsoft/terminal/assets/13099434/5bd4c929-d12a-4246-8055-ec27e215765b)

**Comment by zadjii-msft** [comment]
Pretty sure the launch mode got rolled into the expander below that:
![image](https://github.com/microsoft/terminal/assets/18356694/6dab84d9-0a07-4d96-812e-bc97e1632f0c)

**Reply by iamLazyCode** [reply]
i have never felt so stupid of myself , thankyou so much sir
